apps/condo/domains/organization/integrations/sbbol/sync/syncOrganization.spec.js

```diff
@@ -10,7 +10,8 @@ const { setFakeClientMode, makeLoggedInAdminClient, DATETIME_RE } = require('@op
 const { OrganizationEmployee: OrganizationEmployeeApi, Organization: OrganizationApi } = require('@condo/domains/organization/utils/serverSchema')
 const { registerNewOrganization } = require('@condo/domains/organization/utils/testSchema')
 const { makeClientWithRegisteredOrganization } = require('@condo/domains/organization/utils/testSchema/Organization')
-const { UserAdmin } = require('@condo/domains/user/utils/serverSchema')
+const { USER_FIELDS } = require('@condo/domains/user/gql')
+const { User: UserAPI } = require('@condo/domains/user/utils/serverSchema')
```

Comment:
Why not test utils? They has all fields

Reply:
Server utils uses skipAccess, maybe that's the goal of this tests....idk